libpod/container.go

```diff
@@ -422,14 +446,63 @@ func (c *Container) Start() error {
 	return nil
 }
 
-// Stop stops a container
-func (c *Container) Stop() error {
-	return ErrNotImplemented
+// Stop uses the container's requested stop signal (or SIGTERM if no signal was
```

Comment:
User cn not request stop signal.  The image/container can specify a stop signal.

Reply:
Can you fix this comment?